inferentia/scripts/gen_triton_model.py

```diff
@@ -337,32 +540,50 @@ def create_model_file():
         type=int,
         default=0,
         help='The maximum batch size for the model being generated')
+    parser.add_argument('--model_type',
+                        type=str,
+                        required=True,
```

Comment:
Add a line for choices i.e. `choices=['pytorch', 'tensorflow'],` This allows the parser to validate the options automatically